downstream/modules/platform/proc-inventory-file-setup-rpm.adoc

```diff
@@ -0,0 +1,28 @@
+[id="inventory-file-setup-rpm"]
+
+= Setting up the inventory file
+
+Before upgrading your {PlatformName} installation, edit the `inventory file so that it matches your desired configuration. You can keep the same parameters from your existing Ansible Automation Platform deployment or you can modify the parameters to match any changes to your environment.
```

Comment:
missed attribute in the second sentence here - {PlatformNameShort}

Reply:
Fixed.